src/assignments/tests/assignments.e2e.spec.ts

```diff
@@ -839,6 +925,34 @@ describe('Assignments', () => {
         })
         .expect(403);
     });
+
+    it('should save an image', async () => {
+      const assignment = await formsDbConnection
+        .collection('assignments')
+        .insertOne({
+          ...assignments.defaultAssignment,
+          geostore: assignments.geostore.id,
+          name: 'name',
+          monitors: [ROLES.USER.id],
+          createdBy: ROLES.ADMIN.id,
+        });
+
+      const filename = 'image.png';
+      const fileData = Buffer.from('TestFileContent', 'utf8');
+
+      const response = await request(app.getHttpServer())
+        .patch(`/assignments/${assignment.insertedId.toString()}`)
+        .attach('image', fileData, filename)
+        .set('Authorization', 'ADMIN')
+        .expect(200);
+
+      expect(response.body).toHaveProperty('data');
+      expect(response.body.data).toHaveProperty('attributes');
+      expect(response.body.data.attributes).toHaveProperty(
+        'image',
+        `https://s3.amazonaws.com/bucket/folder/uuid.ext`,
+      );
```

Comment:
I like to do this so that it's easier to see where the test fails (i.e. it has failed on this assertion rather than having to trace "cannot find property blah of undefined")